cropper/app/controllers/Application.scala

```diff
@@ -44,52 +47,24 @@ object Application extends Controller with ArgoHelpers {
 
   def index = Authenticated { indexResponse }
 
+  def export = Authenticated.async(parseExportRequest) { httpRequest =>
+    val exportRequest = httpRequest.body
+    val user = httpRequest.user
 
-  val cropSourceForm: Form[CropSource] = Form(
-    tuple("source" -> nonEmptyText, "x" -> number, "y" -> number, "width" -> number, "height" -> number, "aspectRatio" -> optional(nonEmptyText))
-      .transform[CropSource]({ case (source, x, y, w, h, r) => CropSource(source, Bounds(x, y, w, h), r) },
-                       { case CropSource(source, Bounds(x, y, w, h), r) => (source, x, y, w, h, r) })
-  )
+    executeRequest(exportRequest, user).map { case (imageId, export) =>
+      val cropJson = Json.toJson(export).as[JsObject]
+      val exports  = Json.obj(
+        "id" -> imageId,
+        "data" -> Json.arr(cropJson)
+      )
 
-  def crop = Authenticated.async { httpRequest =>
-
-    val author: Option[String] = httpRequest.user match {
-      case user: AuthenticatedService => Some(user.name)
-      case user: PandaUser => Some(user.email)
+      Notifications.publish(exports, "update-image-exports")
+      Ok(cropJson).as(ArgoMediaType)
+    } recover {
+      case InvalidImage => respondError(BadRequest, "invalid-image", InvalidImage.getMessage)
+      case MissingSecureSourceUrl => respondError(BadRequest, "no-source-image", MissingSecureSourceUrl.getMessage)
+      case InvalidCropRequest => respondError(BadRequest, "invalid-crop", InvalidCropRequest.getMessage)
```

Comment:
Are we going to have a case of the media API being down as that is a dependency?